src/fides/api/task/task_resources.py

```diff
@@ -96,39 +86,42 @@ def close(self) -> None:
 
 
 class TaskResources:
-    """Shared information and environment for all nodes of a given task.
+    """Holds some Database resources for the given task.
     This includes
      - the privacy request
+     - the request task
      - the policy
-     - redis connection
-     -  configurations to any outside resources the task will require to run
+     - configurations to any outside resources the task will require to run
```

Comment:
I'm still using this TaskResources object to slim down the number of variables needed to instantiate a GraphTask - some of the attributes have been moved to GraphTask itself.  Largely this was kept however because we are still supporting DSR 2.0. DSR 2.0 queries a bunch of resources up front and saves them here.  Further, it has a lot of the caching methods.